recipes-bsp/u-boot/u-boot/0004-sun8i-disable-Vendor-Parameter-Protection.patch

```diff
@@ -0,0 +1,28 @@
+From 64f436eea25dc1cf2659931515f3f64e9451f298 Mon Sep 17 00:00:00 2001
+From: Jakub Lecki <jakub.lecki@3mdeb.com>
+Date: Mon, 26 Jul 2021 10:12:28 +0200
+Subject: [PATCH] sun8i: disable Vendor Parameter Protection
+
+by setting CONFIG_ENV_OVERWRITE to 1
+
+Signed-off-by: Jakub Lecki <jakub.lecki@3mdeb.com>
+---
+ include/configs/sun8i.h | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/include/configs/sun8i.h b/include/configs/sun8i.h
+index 9b4675e4c374..228254e2076d 100644
+--- a/include/configs/sun8i.h
++++ b/include/configs/sun8i.h
+@@ -12,6 +12,8 @@
+  * A23 specific configuration
+  */
+ 
++#define CONFIG_ENV_OVERWRITE 1
```

Comment:
@leckijakub A short comment explaining why we are setting this config would be nice

Reply:
@TomaszAIR I have added a comment